### PR TITLE
ts'ified test/directives

### PR DESCRIPTION
**Pull request by ttowncompiled**
[test.unit.js,
test.unit.cjs,
test.unit.dart,
build.dart]
-> all pass

### CODE REVIEW

File: modules/angular2/test/directives/non_bindable_spec.ts [hunk 1]
```diff
@@ -54,20 +59,21 @@ export function main() {
...
-  text: string;
-  constructor() {
-    this.text = 'foo';
-  }
-}
-
 @Directive({
   selector: '[test-dec]'
 })
+@Injectable()
```
Comment (vicb) [inline modules/angular2/test/directives/non_bindable_spec.ts]: Injectable should not be required for Directives (& Components)

**Comment by vicb** [comment]
You need t OK run clang format, check DEVELOPPER or CONTRIBUTING.md (cant remember atm)

**Comment by ttowncompiled** [comment]
I signed it!

**Comment by ttowncompiled** [comment]
I did author those commits.

**Pull request closed without merging**